kedify-agent/templates/agent-deployment.yaml

```
@@ -26,8 +26,16 @@ spec:
     spec:
       containers:
         - args:
-            - --leader-elect
-            - --zap-log-level={{ .Values.agent.logLevel }}
+          - --leader-elect
+          - --zap-log-level={{ default .Values.agent.logLevel .Values.agent.logging.level }}
+          - --zap-log-format={{ .Values.agent.logging.format }}
+          - --zap-log-time-encoding={{ .Values.agent.logging.timeEncoding }}
+          - --zap-log-stack-traces={{ .Values.agent.logging.stackTracesEnabled }}
+          - --zap-encoder={{ .Values.agent.logFormat }}
+          - -zap-time-encoding={{ .Values.agent.logTimeEncoding }}
```

```suggestion
          - --zap-time-encoding={{ .Values.agent.logTimeEncoding }}
```

but there is no `logTimeEncoding` in the values, so it will be empty imho

---

Fixed, PTAL